Sources/StructuredQueriesMacros/SelectionMacro.swift

```diff
@@ -278,7 +278,12 @@ extension SelectionMacro: MemberMacro {
           .text
       }
 
-      allColumns.append((identifier, columnQueryValueType))
+      let defaultValue: ExprSyntax? =
+        binding.initializer.map(\.value.trimmed)
+        // TODO: Revisit this with multi-column support.
+        // ?? (columnQueryValueType?.isOptionalType == true ? ExprSyntax(NilLiteralExprSyntax()) : nil)
```

Comment:
We unfortunately can't automatically do `= #bind(nil)` because outer-joined tables aren't bindable:

```swift
var reminder: Reminder?
```

But if/when we figure out #108 we could revisit this.